### PR TITLE
refactor: status colors should be in status icons

### PR DESCRIPTION
**Pull request by elevatebart**

### CODE REVIEW

File: icon-registry/icons/status-cancelled-outline_x16.svg [hunk 1]
```diff
@@ -1,3 +1,4 @@
-<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M6 8H10M8 14C4.68629 14 2 11.3137 2 8C2 4.68629 4.68629 2 8 2C11.3137 2 14 4.68629 14 8C14 11.3137 11.3137 14 8 14Z" stroke="#AFB3C7" stroke-width="2" stroke-linecap="round"/>
+<svg viewBox="0 0 16 16" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <path d="M6 8H10M8 14C4.68629 14 2 11.3137 2 8C2 4.68629 4.68629 2 8 2C11.3137 2 14 4.68629 14 8C14 11.3137 11.3137 14 8 14Z" fill="currentColor" class="icon-dark" stroke-width="2" stroke-linecap="round"/>
```
Comment (mapsandapps) [inline icon-registry/icons/status-cancelled-outline_x16.svg:3]:
```suggestion
  <path d="M6 8H10M8 14C4.68629 14 2 11.3137 2 8C2 4.68629 4.68629 2 8 2C11.3137 2 14 4.68629 14 8C14 11.3137 11.3137 14 8 14Z" stroke="currentColor" class="icon-dark" stroke-width="2" stroke-linecap="round"/>
```

File: icon-registry/icons/status-cancelled-outline_x24.svg [hunk 2]
```diff
@@ -1,3 +1,4 @@
-<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M8 12H16M12 21C7.02944 21 3 16.9706 3 12C3 7.02944 7.02944 3 12 3C16.9706 3 21 7.02944 21 12C21 16.9706 16.9706 21 12 21Z" stroke="#AFB3C7" stroke-width="2" stroke-linecap="round"/>
+<svg viewBox="0 0 24 24" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <path d="M8 12H16M12 21C7.02944 21 3 16.9706 3 12C3 7.02944 7.02944 3 12 3C16.9706 3 21 7.02944 21 12C21 16.9706 16.9706 21 12 21Z" fill="currentColor" class="icon-dark" stroke-width="2" stroke-linecap="round"/>
```
Comment (mapsandapps) [inline icon-registry/icons/status-cancelled-outline_x24.svg:3]:
```suggestion
  <path d="M8 12H16M12 21C7.02944 21 3 16.9706 3 12C3 7.02944 7.02944 3 12 3C16.9706 3 21 7.02944 21 12C21 16.9706 16.9706 21 12 21Z" stroke="currentColor" class="icon-dark" stroke-width="2" stroke-linecap="round"/>
```

File: icon-registry/icons/status-cancelled-simple_x4.svg [hunk 3]
```diff
@@ -1,3 +1,4 @@
-<svg viewBox="0 0 4 4" fill="none" xmlns="http://www.w3.org/2000/svg">
-<circle cx="2" cy="2" r="1.5" stroke="#AFB3C7"/>
+<svg viewBox="0 0 4 4" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <circle cx="2" cy="2" r="1.5" fill="currentColor" class="icon-dark"/>
```
Comment (mapsandapps) [inline icon-registry/icons/status-cancelled-simple_x4.svg:3]:
```suggestion
  <circle cx="2" cy="2" r="1.5" stroke="currentColor" class="icon-dark"/>
```

File: icon-registry/icons/status-cancelled-simple_x8.svg [hunk 4]
```diff
@@ -1,3 +1,4 @@
-<svg viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M1 4H7" stroke="#AFB3C7" stroke-width="2" stroke-linecap="round"/>
+<svg viewBox="0 0 8 8" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <path d="M1 4H7" fill="currentColor" class="icon-dark" stroke-width="2" stroke-linecap="round"/>
```
Comment (mapsandapps) [inline icon-registry/icons/status-cancelled-simple_x8.svg:3]:
```suggestion
  <path d="M1 4H7" stroke="currentColor" class="icon-dark" stroke-width="2" stroke-linecap="round"/>
```

File: icon-registry/icons/status-cancelled-solid_x12.svg [hunk 5]
```diff
@@ -1,3 +1,4 @@
-<svg viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M2 6L10 6" stroke="#AFB3C7" stroke-width="2" stroke-linecap="round"/>
+<svg viewBox="0 0 12 12" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <path d="M2 6L10 6" fill="currentColor" class="icon-dark" stroke-width="2" stroke-linecap="round"/>
```
Comment (mapsandapps) [inline icon-registry/icons/status-cancelled-solid_x12.svg:3]:
```suggestion
  <path d="M2 6L10 6" stroke="currentColor" class="icon-dark" stroke-width="2" stroke-linecap="round"/>
```

File: icon-registry/icons/status-errored-outline_x24.svg [hunk 6]
```diff
@@ -1,5 +1,6 @@
-<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<circle cx="12" cy="12" r="9" stroke="#DB7903" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M12 7C12.5523 7 13 7.44772 13 8V13C13 13.5523 12.5523 14 12 14C11.4477 14 11 13.5523 11 13V8C11 7.44772 11.4477 7 12 7Z" fill="#E56F4A"/>
-<path d="M13 16C13 16.5523 12.5523 17 12 17C11.4477 17 11 16.5523 11 16C11 15.4477 11.4477 15 12 15C12.5523 15 13 15.4477 13 16Z" fill="#E56F4A"/>
+<svg viewBox="0 0 24 24" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <circle cx="12" cy="12" r="9" fill="currentColor" class="icon-dark" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
```
Comment (mapsandapps) [inline icon-registry/icons/status-errored-outline_x24.svg:3]:
```suggestion
  <circle cx="12" cy="12" r="9" stroke="currentColor" class="icon-dark" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
```

File: icon-registry/icons/status-failed-outline_x16.svg [hunk 7]
```diff
@@ -1,3 +1,4 @@
...
+  xmlns="http://www.w3.org/2000/svg">
+  <path d="M6.70711 5.29289C6.31658 4.90237 5.68342 4.90237 5.29289 5.29289C4.90237 5.68342 4.90237 6.31658 5.29289 6.70711L6.70711 5.29289ZM9.29289 10.7071C9.68342 11.0976 10.3166 11.0976 10.7071 10.7071C11.0976 10.3166 11.0976 9.68342 10.7071 9.29289L9.29289 10.7071ZM10.7071 6.70711C11.0976 6.31658 11.0976 5.68342 10.7071 5.29289C10.3166 4.90237 9.68342 4.90237 9.29289 5.29289L10.7071 6.70711ZM5.29289 9.29289C4.90237 9.68342 4.90237 10.3166 5.29289 10.7071C5.68342 11.0976 6.31658 11.0976 6.70711 10.7071L5.29289 9.29289ZM13 8C13 10.7614 10.7614 13 8 13V15C11.866 15 15 11.866 15 8H13ZM8 13C5.23858 13 3 10.7614 3 8H1C1 11.866 4.13401 15 8 15V13ZM3 8C3 5.23858 5.23858 3 8 3V1C4.13401 1 1 4.13401 1 8H3ZM8 3C10.7614 3 13 5.23858 13 8H15C15 4.13401 11.866 1 8 1V3ZM5.29289 6.70711L7.29289 8.70711L8.70711 7.29289L6.70711 5.29289L5.29289 6.70711ZM7.29289 8.70711L9.29289 10.7071L10.7071 9.29289L8.70711 7.29289L7.29289 8.70711ZM9.29289 5.29289L7.29289 7.29289L8.70711 8.70711L10.7071 6.70711L9.29289 5.29289ZM7.29289 7.29289L5.29289 9.29289L6.70711 10.7071L8.70711 8.70711L7.29289 7.29289Z" fill="#E45770" class="icon-dark"/>
```
Comment (mapsandapps) [inline icon-registry/icons/status-failed-outline_x16.svg:3]: did you not want these failed ones to use currentColor?

**Review by mapsandapps — CHANGES_REQUESTED** [review]
looking good other than a few typos. good idea 😎

File: icon-registry/icons/status-failed-outline_x16.svg (hunk 7 above)
Comment (elevatebart) [reply icon-registry/icons/status-failed-outline_x16.svg:3]: I might have missed that

Comment (elevatebart) [reply icon-registry/icons/status-failed-outline_x16.svg:3]:
```suggestion
  <path d="M6.70711 5.29289C6.31658 4.90237 5.68342 4.90237 5.29289 5.29289C4.90237 5.68342 4.90237 6.31658 5.29289 6.70711L6.70711 5.29289ZM9.29289 10.7071C9.68342 11.0976 10.3166 11.0976 10.7071 10.7071C11.0976 10.3166 11.0976 9.68342 10.7071 9.29289L9.29289 10.7071ZM10.7071 6.70711C11.0976 6.31658 11.0976 5.68342 10.7071 5.29289C10.3166 4.90237 9.68342 4.90237 9.29289 5.29289L10.7071 6.70711ZM5.29289 9.29289C4.90237 9.68342 4.90237 10.3166 5.29289 10.7071C5.68342 11.0976 6.31658 11.0976 6.70711 10.7071L5.29289 9.29289ZM13 8C13 10.7614 10.7614 13 8 13V15C11.866 15 15 11.866 15 8H13ZM8 13C5.23858 13 3 10.7614 3 8H1C1 11.866 4.13401 15 8 15V13ZM3 8C3 5.23858 5.23858 3 8 3V1C4.13401 1 1 4.13401 1 8H3ZM8 3C10.7614 3 13 5.23858 13 8H15C15 4.13401 11.866 1 8 1V3ZM5.29289 6.70711L7.29289 8.70711L8.70711 7.29289L6.70711 5.29289L5.29289 6.70711ZM7.29289 8.70711L9.29289 10.7071L10.7071 9.29289L8.70711 7.29289L7.29289 8.70711ZM9.29289 5.29289L7.29289 7.29289L8.70711 8.70711L10.7071 6.70711L9.29289 5.29289ZM7.29289 7.29289L5.29289 9.29289L6.70711 10.7071L8.70711 8.70711L7.29289 7.29289Z" fill="currentColor" class="icon-dark"/>
```

File: icon-registry/icons/status-failed-outline_x24.svg [hunk 8]
```diff
@@ -1,3 +1,4 @@
...
+  xmlns="http://www.w3.org/2000/svg">
+  <path d="M9.70711 8.29289C9.31658 7.90237 8.68342 7.90237 8.29289 8.29289C7.90237 8.68342 7.90237 9.31658 8.29289 9.70711L9.70711 8.29289ZM14.2929 15.7071C14.6834 16.0976 15.3166 16.0976 15.7071 15.7071C16.0976 15.3166 16.0976 14.6834 15.7071 14.2929L14.2929 15.7071ZM15.7071 9.70711C16.0976 9.31658 16.0976 8.68342 15.7071 8.29289C15.3166 7.90237 14.6834 7.90237 14.2929 8.29289L15.7071 9.70711ZM8.29289 14.2929C7.90237 14.6834 7.90237 15.3166 8.29289 15.7071C8.68342 16.0976 9.31658 16.0976 9.70711 15.7071L8.29289 14.2929ZM8.29289 9.70711L11.2929 12.7071L12.7071 11.2929L9.70711 8.29289L8.29289 9.70711ZM11.2929 12.7071L14.2929 15.7071L15.7071 14.2929L12.7071 11.2929L11.2929 12.7071ZM14.2929 8.29289L11.2929 11.2929L12.7071 12.7071L15.7071 9.70711L14.2929 8.29289ZM11.2929 11.2929L8.29289 14.2929L9.70711 15.7071L12.7071 12.7071L11.2929 11.2929ZM20 12C20 16.4183 16.4183 20 12 20V22C17.5228 22 22 17.5228 22 12H20ZM12 20C7.58172 20 4 16.4183 4 12H2C2 17.5228 6.47715 22 12 22V20ZM4 12C4 7.58172 7.58172 4 12 4V2C6.47715 2 2 6.47715 2 12H4ZM12 4C16.4183 4 20 7.58172 20 12H22C22 6.47715 17.5228 2 12 2V4Z" fill="#E45770" class="icon-dark"/>
```
Comment (elevatebart) [inline icon-registry/icons/status-failed-outline_x24.svg:3]:
```suggestion
  <path d="M9.70711 8.29289C9.31658 7.90237 8.68342 7.90237 8.29289 8.29289C7.90237 8.68342 7.90237 9.31658 8.29289 9.70711L9.70711 8.29289ZM14.2929 15.7071C14.6834 16.0976 15.3166 16.0976 15.7071 15.7071C16.0976 15.3166 16.0976 14.6834 15.7071 14.2929L14.2929 15.7071ZM15.7071 9.70711C16.0976 9.31658 16.0976 8.68342 15.7071 8.29289C15.3166 7.90237 14.6834 7.90237 14.2929 8.29289L15.7071 9.70711ZM8.29289 14.2929C7.90237 14.6834 7.90237 15.3166 8.29289 15.7071C8.68342 16.0976 9.31658 16.0976 9.70711 15.7071L8.29289 14.2929ZM8.29289 9.70711L11.2929 12.7071L12.7071 11.2929L9.70711 8.29289L8.29289 9.70711ZM11.2929 12.7071L14.2929 15.7071L15.7071 14.2929L12.7071 11.2929L11.2929 12.7071ZM14.2929 8.29289L11.2929 11.2929L12.7071 12.7071L15.7071 9.70711L14.2929 8.29289ZM11.2929 11.2929L8.29289 14.2929L9.70711 15.7071L12.7071 12.7071L11.2929 11.2929ZM20 12C20 16.4183 16.4183 20 12 20V22C17.5228 22 22 17.5228 22 12H20ZM12 20C7.58172 20 4 16.4183 4 12H2C2 17.5228 6.47715 22 12 22V20ZM4 12C4 7.58172 7.58172 4 12 4V2C6.47715 2 2 6.47715 2 12H4ZM12 4C16.4183 4 20 7.58172 20 12H22C22 6.47715 17.5228 2 12 2V4Z" fill="currentColor" class="icon-dark"/>
```

File: icon-registry/icons/status-failed-simple_x12.svg [hunk 9]
```diff
@@ -1,3 +1,4 @@
...
+<svg viewBox="0 0 12 12" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <path d="M3.70711 2.29289C3.31658 1.90237 2.68342 1.90237 2.29289 2.29289C1.90237 2.68342 1.90237 3.31658 2.29289 3.70711L3.70711 2.29289ZM8.29289 9.70711C8.68342 10.0976 9.31658 10.0976 9.70711 9.70711C10.0976 9.31658 10.0976 8.68342 9.70711 8.29289L8.29289 9.70711ZM9.70711 3.70711C10.0976 3.31658 10.0976 2.68342 9.70711 2.29289C9.31658 1.90237 8.68342 1.90237 8.29289 2.29289L9.70711 3.70711ZM2.29289 8.29289C1.90237 8.68342 1.90237 9.31658 2.29289 9.70711C2.68342 10.0976 3.31658 10.0976 3.70711 9.70711L2.29289 8.29289ZM2.29289 3.70711L8.29289 9.70711L9.70711 8.29289L3.70711 2.29289L2.29289 3.70711ZM8.29289 2.29289L2.29289 8.29289L3.70711 9.70711L9.70711 3.70711L8.29289 2.29289Z" fill="#E45770" class="icon-dark"/>
```
Comment (elevatebart) [inline icon-registry/icons/status-failed-simple_x12.svg:3]:
```suggestion
  <path d="M3.70711 2.29289C3.31658 1.90237 2.68342 1.90237 2.29289 2.29289C1.90237 2.68342 1.90237 3.31658 2.29289 3.70711L3.70711 2.29289ZM8.29289 9.70711C8.68342 10.0976 9.31658 10.0976 9.70711 9.70711C10.0976 9.31658 10.0976 8.68342 9.70711 8.29289L8.29289 9.70711ZM9.70711 3.70711C10.0976 3.31658 10.0976 2.68342 9.70711 2.29289C9.31658 1.90237 8.68342 1.90237 8.29289 2.29289L9.70711 3.70711ZM2.29289 8.29289C1.90237 8.68342 1.90237 9.31658 2.29289 9.70711C2.68342 10.0976 3.31658 10.0976 3.70711 9.70711L2.29289 8.29289ZM2.29289 3.70711L8.29289 9.70711L9.70711 8.29289L3.70711 2.29289L2.29289 3.70711ZM8.29289 2.29289L2.29289 8.29289L3.70711 9.70711L9.70711 3.70711L8.29289 2.29289Z" fill="currentColor" class="icon-dark"/>
```

File: icon-registry/icons/status-failed-simple_x16.svg [hunk 10]
```diff
@@ -1,3 +1,4 @@
...
+<svg viewBox="0 0 16 16" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <path d="M4.70711 3.29289C4.31658 2.90237 3.68342 2.90237 3.29289 3.29289C2.90237 3.68342 2.90237 4.31658 3.29289 4.70711L4.70711 3.29289ZM11.2929 12.7071C11.6834 13.0976 12.3166 13.0976 12.7071 12.7071C13.0976 12.3166 13.0976 11.6834 12.7071 11.2929L11.2929 12.7071ZM12.7071 4.70711C13.0976 4.31658 13.0976 3.68342 12.7071 3.29289C12.3166 2.90237 11.6834 2.90237 11.2929 3.29289L12.7071 4.70711ZM3.29289 11.2929C2.90237 11.6834 2.90237 12.3166 3.29289 12.7071C3.68342 13.0976 4.31658 13.0976 4.70711 12.7071L3.29289 11.2929ZM3.29289 4.70711L7.29289 8.70711L8.70711 7.29289L4.70711 3.29289L3.29289 4.70711ZM7.29289 8.70711L11.2929 12.7071L12.7071 11.2929L8.70711 7.29289L7.29289 8.70711ZM11.2929 3.29289L7.29289 7.29289L8.70711 8.70711L12.7071 4.70711L11.2929 3.29289ZM7.29289 7.29289L3.29289 11.2929L4.70711 12.7071L8.70711 8.70711L7.29289 7.29289Z" fill="#E45770" class="icon-dark"/>
```
Comment (elevatebart) [inline icon-registry/icons/status-failed-simple_x16.svg:3]:
```suggestion
  <path d="M4.70711 3.29289C4.31658 2.90237 3.68342 2.90237 3.29289 3.29289C2.90237 3.68342 2.90237 4.31658 3.29289 4.70711L4.70711 3.29289ZM11.2929 12.7071C11.6834 13.0976 12.3166 13.0976 12.7071 12.7071C13.0976 12.3166 13.0976 11.6834 12.7071 11.2929L11.2929 12.7071ZM12.7071 4.70711C13.0976 4.31658 13.0976 3.68342 12.7071 3.29289C12.3166 2.90237 11.6834 2.90237 11.2929 3.29289L12.7071 4.70711ZM3.29289 11.2929C2.90237 11.6834 2.90237 12.3166 3.29289 12.7071C3.68342 13.0976 4.31658 13.0976 4.70711 12.7071L3.29289 11.2929ZM3.29289 4.70711L7.29289 8.70711L8.70711 7.29289L4.70711 3.29289L3.29289 4.70711ZM7.29289 8.70711L11.2929 12.7071L12.7071 11.2929L8.70711 7.29289L7.29289 8.70711ZM11.2929 3.29289L7.29289 7.29289L8.70711 8.70711L12.7071 4.70711L11.2929 3.29289ZM7.29289 7.29289L3.29289 11.2929L4.70711 12.7071L8.70711 8.70711L7.29289 7.29289Z" fill="currentColor" class="icon-dark"/>
```

File: icon-registry/icons/status-failed-simple_x24.svg [hunk 11]
```diff
@@ -1,3 +1,4 @@
...
+<svg viewBox="0 0 24 24" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <path d="M6.70711 5.29289C6.31658 4.90237 5.68342 4.90237 5.29289 5.29289C4.90237 5.68342 4.90237 6.31658 5.29289 6.70711L6.70711 5.29289ZM17.2929 18.7071C17.6834 19.0976 18.3166 19.0976 18.7071 18.7071C19.0976 18.3166 19.0976 17.6834 18.7071 17.2929L17.2929 18.7071ZM18.7071 6.70711C19.0976 6.31658 19.0976 5.68342 18.7071 5.29289C18.3166 4.90237 17.6834 4.90237 17.2929 5.29289L18.7071 6.70711ZM5.29289 17.2929C4.90237 17.6834 4.90237 18.3166 5.29289 18.7071C5.68342 19.0976 6.31658 19.0976 6.70711 18.7071L5.29289 17.2929ZM5.29289 6.70711L11.2929 12.7071L12.7071 11.2929L6.70711 5.29289L5.29289 6.70711ZM11.2929 12.7071L17.2929 18.7071L18.7071 17.2929L12.7071 11.2929L11.2929 12.7071ZM17.2929 5.29289L11.2929 11.2929L12.7071 12.7071L18.7071 6.70711L17.2929 5.29289ZM11.2929 11.2929L5.29289 17.2929L6.70711 18.7071L12.7071 12.7071L11.2929 11.2929Z" fill="#E45770" class="icon-dark"/>
```
Comment (elevatebart) [inline icon-registry/icons/status-failed-simple_x24.svg:3]:
```suggestion
  <path d="M6.70711 5.29289C6.31658 4.90237 5.68342 4.90237 5.29289 5.29289C4.90237 5.68342 4.90237 6.31658 5.29289 6.70711L6.70711 5.29289ZM17.2929 18.7071C17.6834 19.0976 18.3166 19.0976 18.7071 18.7071C19.0976 18.3166 19.0976 17.6834 18.7071 17.2929L17.2929 18.7071ZM18.7071 6.70711C19.0976 6.31658 19.0976 5.68342 18.7071 5.29289C18.3166 4.90237 17.6834 4.90237 17.2929 5.29289L18.7071 6.70711ZM5.29289 17.2929C4.90237 17.6834 4.90237 18.3166 5.29289 18.7071C5.68342 19.0976 6.31658 19.0976 6.70711 18.7071L5.29289 17.2929ZM5.29289 6.70711L11.2929 12.7071L12.7071 11.2929L6.70711 5.29289L5.29289 6.70711ZM11.2929 12.7071L17.2929 18.7071L18.7071 17.2929L12.7071 11.2929L11.2929 12.7071ZM17.2929 5.29289L11.2929 11.2929L12.7071 12.7071L18.7071 6.70711L17.2929 5.29289ZM11.2929 11.2929L5.29289 17.2929L6.70711 18.7071L12.7071 12.7071L11.2929 11.2929Z" fill="currentColor" class="icon-dark"/>
```

File: icon-registry/icons/status-failed-simple_x4.svg [hunk 12]
```diff
@@ -1,3 +1,4 @@
-<svg viewBox="0 0 4 4" fill="none" xmlns="http://www.w3.org/2000/svg">
-<circle cx="2" cy="2" r="2" fill="#E45770"/>
+<svg viewBox="0 0 4 4" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <circle cx="2" cy="2" r="2" fill="#E45770" class="icon-dark"/>
```
Comment (elevatebart) [inline icon-registry/icons/status-failed-simple_x4.svg:3]:
```suggestion
  <circle cx="2" cy="2" r="2" fill="currentColor" class="icon-dark"/>
```

File: icon-registry/icons/status-failed-simple_x8.svg [hunk 13]
```diff
@@ -1,3 +1,4 @@
...
+<svg viewBox="0 0 8 8" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <path d="M2.70711 1.29289C2.31658 0.902369 1.68342 0.902369 1.29289 1.29289C0.902369 1.68342 0.902369 2.31658 1.29289 2.70711L2.70711 1.29289ZM5.29289 6.70711C5.68342 7.09763 6.31658 7.09763 6.70711 6.70711C7.09763 6.31658 7.09763 5.68342 6.70711 5.29289L5.29289 6.70711ZM6.70711 2.70711C7.09763 2.31658 7.09763 1.68342 6.70711 1.29289C6.31658 0.902369 5.68342 0.902369 5.29289 1.29289L6.70711 2.70711ZM1.29289 5.29289C0.902369 5.68342 0.902369 6.31658 1.29289 6.70711C1.68342 7.09763 2.31658 7.09763 2.70711 6.70711L1.29289 5.29289ZM1.29289 2.70711L5.29289 6.70711L6.70711 5.29289L2.70711 1.29289L1.29289 2.70711ZM5.29289 1.29289L1.29289 5.29289L2.70711 6.70711L6.70711 2.70711L5.29289 1.29289Z" fill="#E45770" class="icon-dark"/>
```
Comment (elevatebart) [inline icon-registry/icons/status-failed-simple_x8.svg:3]:
```suggestion
  <path d="M2.70711 1.29289C2.31658 0.902369 1.68342 0.902369 1.29289 1.29289C0.902369 1.68342 0.902369 2.31658 1.29289 2.70711L2.70711 1.29289ZM5.29289 6.70711C5.68342 7.09763 6.31658 7.09763 6.70711 6.70711C7.09763 6.31658 7.09763 5.68342 6.70711 5.29289L5.29289 6.70711ZM6.70711 2.70711C7.09763 2.31658 7.09763 1.68342 6.70711 1.29289C6.31658 0.902369 5.68342 0.902369 5.29289 1.29289L6.70711 2.70711ZM1.29289 5.29289C0.902369 5.68342 0.902369 6.31658 1.29289 6.70711C1.68342 7.09763 2.31658 7.09763 2.70711 6.70711L1.29289 5.29289ZM1.29289 2.70711L5.29289 6.70711L6.70711 5.29289L2.70711 1.29289L1.29289 2.70711ZM5.29289 1.29289L1.29289 5.29289L2.70711 6.70711L6.70711 2.70711L5.29289 1.29289Z" fill="currentColor" class="icon-dark"/>
```

File: icon-registry/icons/status-failed-solid_x16.svg [hunk 14]
```diff
@@ -1,4 +1,5 @@
-<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-<circle cx="8" cy="8" r="7" fill="#E45770"/>
-<path d="M6.70711 5.29289C6.31658 4.90237 5.68342 4.90237 5.29289 5.29289C4.90237 5.68342 4.90237 6.31658 5.29289 6.70711L6.70711 5.29289ZM9.29289 10.7071C9.68342 11.0976 10.3166 11.0976 10.7071 10.7071C11.0976 10.3166 11.0976 9.68342 10.7071 9.29289L9.29289 10.7071ZM10.7071 6.70711C11.0976 6.31658 11.0976 5.68342 10.7071 5.29289C10.3166 4.90237 9.68342 4.90237 9.29289 5.29289L10.7071 6.70711ZM5.29289 9.29289C4.90237 9.68342 4.90237 10.3166 5.29289 10.7071C5.68342 11.0976 6.31658 11.0976 6.70711 10.7071L5.29289 9.29289ZM5.29289 6.70711L9.29289 10.7071L10.7071 9.29289L6.70711 5.29289L5.29289 6.70711ZM9.29289 5.29289L5.29289 9.29289L6.70711 10.7071L10.7071 6.70711L9.29289 5.29289Z" fill="white"/>
+<svg viewBox="0 0 16 16" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <circle cx="8" cy="8" r="7" fill="#E45770" class="icon-dark"/>
```
Comment (elevatebart) [inline icon-registry/icons/status-failed-solid_x16.svg:3]:
```suggestion
  <circle cx="8" cy="8" r="7" fill="currentColor" class="icon-dark"/>
```

File: icon-registry/icons/status-failed-solid_x24.svg [hunk 15]
```diff
@@ -1,3 +1,4 @@
...
+<svg viewBox="0 0 24 24" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <path fill-rule="evenodd" clip-rule="evenodd" d="M12 22C17.5228 22 22 17.5228 22 12C22 6.47715 17.5228 2 12 2C6.47715 2 2 6.47715 2 12C2 17.5228 6.47715 22 12 22ZM9.70711 8.29289C9.31658 7.90237 8.68342 7.90237 8.29289 8.29289C7.90237 8.68342 7.90237 9.31658 8.29289 9.70711L10.5858 12L8.29289 14.2929C7.90237 14.6834 7.90237 15.3166 8.29289 15.7071C8.68342 16.0976 9.31658 16.0976 9.70711 15.7071L12 13.4142L14.2929 15.7071C14.6834 16.0976 15.3166 16.0976 15.7071 15.7071C16.0976 15.3166 16.0976 14.6834 15.7071 14.2929L13.4142 12L15.7071 9.70711C16.0976 9.31658 16.0976 8.68342 15.7071 8.29289C15.3166 7.90237 14.6834 7.90237 14.2929 8.29289L12 10.5858L9.70711 8.29289Z" fill="#E45770" class="icon-dark"/>
```
Comment (elevatebart) [inline icon-registry/icons/status-failed-solid_x24.svg:3]:
```suggestion
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12 22C17.5228 22 22 17.5228 22 12C22 6.47715 17.5228 2 12 2C6.47715 2 2 6.47715 2 12C2 17.5228 6.47715 22 12 22ZM9.70711 8.29289C9.31658 7.90237 8.68342 7.90237 8.29289 8.29289C7.90237 8.68342 7.90237 9.31658 8.29289 9.70711L10.5858 12L8.29289 14.2929C7.90237 14.6834 7.90237 15.3166 8.29289 15.7071C8.68342 16.0976 9.31658 16.0976 9.70711 15.7071L12 13.4142L14.2929 15.7071C14.6834 16.0976 15.3166 16.0976 15.7071 15.7071C16.0976 15.3166 16.0976 14.6834 15.7071 14.2929L13.4142 12L15.7071 9.70711C16.0976 9.31658 16.0976 8.68342 15.7071 8.29289C15.3166 7.90237 14.6834 7.90237 14.2929 8.29289L12 10.5858L9.70711 8.29289Z" fill="currentColor" class="icon-dark"/>
```

File: icon-registry/icons/status-pending-simple_x8.svg [hunk 16]
```diff
@@ -1,10 +1,11 @@
-<svg fill="none" xmlns="http://www.w3.org/2000/svg">
-<g clip-path="url(#clip0_2588_3918)">
-<path d="M2.85194 1.22839C2.14262 1.52221 1.54538 2.08662 1.22835 2.85198C0.594302 4.38272 1.32121 6.13762 2.85194 6.77167C4.38267 7.40572 6.13758 6.67882 6.77163 5.14808C7.08866 4.38272 7.06544 3.56131 6.77163 2.85198C6.47782 2.14266 5.91341 1.54542 5.14804 1.22839" stroke="#BFC2D4" stroke-width="2"/>
-</g>
-<defs>
-<clipPath id="clip0_2588_3918">
-<rect width="8" height="8" fill="white"/>
-</clipPath>
-</defs>
+<svg fill="none"
```
Comment (elevatebart) [inline icon-registry/icons/status-pending-simple_x8.svg:1]:
```suggestion
<svg viewBox="0 0 8 8" fill="none"
```

File: icon-registry/icons/status-errored-outline_x24.svg [hunk 17]
```diff
@@ -1,5 +1,6 @@
-<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<circle cx="12" cy="12" r="9" stroke="#DB7903" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M12 7C12.5523 7 13 7.44772 13 8V13C13 13.5523 12.5523 14 12 14C11.4477 14 11 13.5523 11 13V8C11 7.44772 11.4477 7 12 7Z" fill="#E56F4A"/>
-<path d="M13 16C13 16.5523 12.5523 17 12 17C11.4477 17 11 16.5523 11 16C11 15.4477 11.4477 15 12 15C12.5523 15 13 15.4477 13 16Z" fill="#E56F4A"/>
+<svg viewBox="0 0 24 24" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <circle cx="12" cy="12" r="9" stroke="currentColor" class="icon-dark" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+  <path fill-rule="evenodd" clip-rule="evenodd" d="M12 7C12.5523 7 13 7.44772 13 8V13C13 13.5523 12.5523 14 12 14C11.4477 14 11 13.5523 11 13V8C11 7.44772 11.4477 7 12 7Z" fill="#E56F4A"/>
+  <path d="M13 16C13 16.5523 12.5523 17 12 17C11.4477 17 11 16.5523 11 16C11 15.4477 11.4477 15 12 15C12.5523 15 13 15.4477 13 16Z" fill="#E56F4A"/>
```
Comment (mapsandapps) [inline icon-registry/icons/status-errored-outline_x24.svg:5]:
```suggestion
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12 7C12.5523 7 13 7.44772 13 8V13C13 13.5523 12.5523 14 12 14C11.4477 14 11 13.5523 11 13V8C11 7.44772 11.4477 7 12 7Z" fill="currentColor"/>
  <path d="M13 16C13 16.5523 12.5523 17 12 17C11.4477 17 11 16.5523 11 16C11 15.4477 11.4477 15 12 15C12.5523 15 13 15.4477 13 16Z" fill="currentColor"/>
```

oops, this icon was wrong before this PR. all the oranges here should have been the same shade

**Review by mapsandapps — CHANGES_REQUESTED** [review]
whoops, just one more

**Review by mapsandapps — APPROVED** [review]

**Comment by elevatebart** [comment]
Now that is the icon you waited for for so long @emilyrohrbough

Sorry it took so long

**Pull request merged**